src/TestingFramework/Bootstrap/AbstractBootstrap.php

```diff
@@ -47,8 +47,10 @@ protected function includeAndStartCoreBootstrap()
     {
         $classLoaderFilepath = $this->getClassLoaderFilepath();
         $classLoader = require $classLoaderFilepath;
+        $classAliasLoader = new \TYPO3\ClassAliasLoader\ClassAliasLoader($classLoader);
 
         SystemEnvironmentBuilder::run(0, SystemEnvironmentBuilder::REQUESTTYPE_BE | SystemEnvironmentBuilder::REQUESTTYPE_CLI);
+        ClassAliasMap::setClassAliasLoader($classAliasLoader);
```

Comment:
Both calls actually happen on Composer level, once the `autoload.php` file is included.
My question therefore is: How exactly is your system set up, that this does not happen?

Reply:
checked with a repo and it is exactly how I suspected. This code isn't needed in a default composer setup with nimut/testing-framework